qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/AbstractRestartUpgradeTestCase.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.bwc;
+
+import org.opensearch.common.settings.Settings;
+import org.opensearch.knn.KNNRestTestCase;
+import org.opensearch.test.rest.OpenSearchRestTestCase;
+
+import static org.opensearch.knn.TestUtils.*;
+
+public abstract class AbstractRestartUpgradeTestCase extends KNNRestTestCase {
+
+    @Override
+    protected final boolean preserveIndicesUponCompletion() {
+        return true;
+    }
+
+    @Override
+    protected final boolean preserveReposUponCompletion() {
+        return true;
+    }
+
+    @Override
+    protected boolean preserveTemplatesUponCompletion() {
+        return true;
+    }
+
+    @Override
+    protected final Settings restClientSettings() {
+        return Settings.builder()
+            .put(super.restClientSettings())
+            // increase the timeout here to 90 seconds to handle long waits for a green
+            // cluster health. the waits for green need to be longer than a minute to
+            // account for delayed shards
+            .put(OpenSearchRestTestCase.CLIENT_SOCKET_TIMEOUT, SECONDS_90)
```

Comment:
IMO, you can call the constant as CLIENT_TIMEOUT_VALUE

Reply:
Why constants are not defined in this Class?

Reply:
updated